gf-guide/metrics.md

```diff
@@ -66,37 +78,30 @@ The Microsoft [OpenType specification](https://www.microsoft.com/typography/otsp
 
 This rule can be avoided if a font is being upgraded and previously had inconsistent values.
 
-**8. LineGap values must be 0.**
+## 8. LineGap values must be 0
 
 The `LineGap` value is a space added to the line height created by the union of the `(typo/hhea)Ascender` and `(typo/hhea)Descender`. It is handled differently according to the environment. This leading value will be added above the text line in most desktop apps. It will be shared above *and* under in web browsers, and ignored in Windows if `Use_Typo_Metrics` is disabled. For better linespacing consistency across platforms, `(typo/hhea)LineGap` values must be `0`.
 
-**9. Uppercases should be centered if the font’s primary script has uppercases letterforms such as Latin, Greek and Cyrillic.**
+#### 9. Uppercases should be centered if the font’s primary script has uppercases letterforms such as Latin, Greek and Cyrillic
 
 Web designers will thank you if you managed to have the same space above and under capitals: `typoAscender - CapsHeight = abs(typoDescender)`. It will make easier for them the setting of padding in buttons for example.
 
-**10. typo/hheaAscender value should be greater than Agrave's yMax when it makes sense.**
+#### 10. typo/hheaAscender value should leave open room for stacked diacritics.
+
+Our minimal required Glyphset for any new family is GF Latin Core, which includes Vietnamese support. Following recent tests to confirm our VM policies, we now require that the typo/hheaAscender be equal to or slightly higher than `A breve acute`. To guarantee uniform positioning across the entire font family, you must use the tallest `A breve acute` as the reference point.
+
+Even if, for any particular reason, the font does not currently support Vietnamese yet, we strongly suggest estimating the ascenders value foreseeing this glyph height for two main reasons
 
-Some Mac applications such as TextEdit will position the first line of text by, either, using the height of the `A grave`, or by using the font’s `hheaAscender` (whichever is taller). To keep the positioning consistent across a family, we require that the `hheaAscender` is greater than the tallest `A grave` in the family. See this issue for further info, <https://github.com/googlefonts/fontbakery/issues/3170>.
+- Once the font is published, it is not possible to modify the vertical metric values. By implementing the above approach, we can keep the option of updating the font in the future to support additional languages.
+- Certain Mac applications, including TextEdit, determine the height of the first line of text by either the `A grave` height or the font's `hheaAscender`, whichever is taller (for additional information on this topic, please refer to this issue: <https://github.com/googlefonts/fontbakery/issues/3170>). So using the suggested `A breve acute` glyph height will prevent clipping in such applications.
```

Comment:
I would say something like: "Acute is a required minimum, and Abreveacute is a strongly recommended minimum" -> this would determine a FAIL and a WARN

Reply:
Actually, now that we require Vietnamese by default (given it is included in the Latin Core) the whole idea is to ensure the line-height works ok also for that language. So I think we should say:
"Abreveacute is a required minimum, and Agrave is a highly recommended minimum"

Reply:
When was it included in Latin_Core?

Reply:
My bad. I was thinking of the minimum requirement for commissioned projects actually

Reply:
@RosaWagner, I've updated the file with the feedback. Please take another look.